### PR TITLE
[MIG] hr_employee_birth_name: Migrated to v12

### PR DESCRIPTION
**Pull request by luistorresm**

### CODE REVIEW

File: hr_employee_birth_name/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# copyright  2017 Denis Leemann, Camptocamp SA
```
Comment (nikul-serpentcs) [inline hr_employee_birth_name/__init__.py]: Remove Copyright from all `__init__` file.

File: hr_employee_birth_name/views/hr_employee_view.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="view_employee_form_inherit_extension" model="ir.ui.view">
+      <field name="name">view.employee.form.inherit.extension</field>
+      <field name="model">hr.employee</field>
+      <field name="inherit_id" ref="hr.view_employee_form"/>
+      <field name="arch" type="xml">
+
+        <xpath expr="//field[@name='marital']" position="after">
```
Comment (nikul-serpentcs) [inline hr_employee_birth_name/views/hr_employee_view.xml]: Direct use `field`

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

File: hr_employee_birth_name/views/hr_employee_view.xml (hunk 2 above)
Comment (luistorresm) [reply hr_employee_birth_name/views/hr_employee_view.xml]: Excuse me, I do not understand this change, you prefer that I remove the xpath?

Regards

Comment (nikul-serpentcs) [reply hr_employee_birth_name/views/hr_employee_view.xml]: Yes remove `xpath` and use `<field name="marital" position="after">`

**Comment by luistorresm** [comment]
Hello @nikul-serpentcs
 
Done.

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

**Review by antonioburic — APPROVED** [review]
Code Review LGTM, installed on the latest master branch instance - everything good ^^

**Comment by lfreeke** [comment]
:+1: (test)

**Review by tafaRU — CHANGES_REQUESTED** [review]
Could you take the occasion to regenerate README file by using `oca-gen-addon-readme`?

**Comment by luistorresm** [comment]
Done @tafaRU 

Regards

**Comment by pedrobaeza** [comment]
Travis is red

File: hr_employee_birth_name/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# copyright  2017 Denis Leemann, Camptocamp SA
```
Comment (nikul-serpentcs) [inline hr_employee_birth_name/models/__init__.py:1]: @luistorresm Remove Copyright from all `__init__`

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

**Comment by tafaRU** [comment]
I'm restarting Travis build as the current errors are as `fatal: Remote branch 12.0 not found in upstream origin`.

File: hr_employee_birth_name/readme/DESCRIPTION.rst [hunk 4]
```diff
@@ -0,0 +1,3 @@
+This module adds the following to employees :
```
Comment (tafaRU) [inline hr_employee_birth_name/readme/DESCRIPTION.rst:1]: Coul you take the opportunity to extend it a bit?

File: hr_employee_birth_name/readme/INSTALL.rst [hunk 5]
```diff
@@ -0,0 +1,8 @@
+To install this module, you need to:
```
Comment (tafaRU) [inline hr_employee_birth_name/readme/INSTALL.rst:1]: I'd avoid this as it's quite useless.

**Review by tafaRU — CHANGES_REQUESTED** [review]
nitpicking, otherwise LGTM

**Review by gurneyalex — APPROVED** [review]

File: hr_employee_birth_name/readme/DESCRIPTION.rst (hunk 4 above)
Comment (luistorresm) [reply hr_employee_birth_name/readme/DESCRIPTION.rst:1]: Hello @tafaRU 

This is a migration, that could be improved in other PR.

**Pull request merged**